src/buyer/buyer.service.spec.ts

```diff
@@ -1,19 +1,19 @@
-import { Test, TestingModule } from '@nestjs/testing';
+// import { Test, TestingModule } from '@nestjs/testing';
```

Comment:
Better to uncomment this file and change UserService into BuyerService for future use